### ISSUE TITLE
CustomSanitizer called twice

### ISSUE THREAD

**Issue by amberlionk** [express-validator/express-validator#584]
```
const schema={
  TestField:{
    customSanitizer: {
      options: (value) => {
        let d=new Date();
        console.log(d.getUTCMilliseconds()+"   "+value);
        return value;
      }
    },
  },
}
```

in server log i see
```
351  aaaaaa
352  aaaaaa
```

node -v  = ``v8.10.0``
express-validator@5.2.0

**Comment by orlavy-bigabid** [comment]
It is a problem which caused a lot of confusion and weird problems for me also.
In my opinion, the sanitizer should be called only once and in its proper location in the chain

**Comment by gustavohenke** [comment]
Shipped on [v5.3.0](https://github.com/express-validator/express-validator/releases/tag/v5.3.0)! 🚢
